app/javascript/components/shared/Capitalize.jsx

```diff
@@ -0,0 +1,5 @@
+const Capitalize = (str) => {
```

Comment:
@excelsior I agree, but not at all. Why do you think it's not a component? I rather use Lodash, which I will now, but I was thinking of avoiding the import of such a library only to use "capitalize".

I will change it anyway, because I know there's a bunch of powerful methods that I can use in there.